packages/memory-inspector/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@theia/memory-inspector",
+  "version": "1.27.0",
+  "description": "Theia - Memory Inspector",
+  "keywords": [
+    "theia-extension"
+  ],
+  "homepage": "https://github.com/eclipse-theia/theia",
+  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-theia/theia.git"
+  },
+  "bugs": {
+    "url": "https://github.com/eclipse-theia/theia/issues"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "scripts": {
+    "build": "theiaext build",
+    "clean": "theiaext clean",
+    "compile": "theiaext compile",
+    "lint": "theiaext lint",
+    "test": "theiaext test",
+    "watch": "theiaext watch"
+  },
+  "dependencies": {
+    "@theia/core": "1.27.0",
+    "@theia/debug": "1.27.0",
+    "long": "^4.0.0",
+    "vscode-debugprotocol": "^1.48.0"
```

Comment:
Do we need to also update `vscode-debugprotocol` in `@theia/debug` and `@theia/plugin-ext` to follow this version:

https://github.com/eclipse-theia/theia/blob/b000f9603107cf20e46cb989505b1e2dbb546808/packages/debug/package.json#L25

https://github.com/eclipse-theia/theia/blob/b000f9603107cf20e46cb989505b1e2dbb546808/packages/plugin-ext/package.json#L45

Reply:
Because they're both caret versions, we're currently getting a high-enough version (1.51) in our `yarn.lock`, but it's probably best to be explicit.

Reply:
I've updated the entries in our various `package.json`. They've deprecated this package name, so I've updated that, as well, with the unfortunate consequence that the new package name isn't part of the 3PP baseline, apparently.